Dockerfile

```diff
@@ -4,10 +4,10 @@ MAINTAINER Eyevinn Technology <work@eyevinn.se>
 WORKDIR /usr/local/bin
 
 RUN apk add --no-cache --update openssl curl ca-certificates && \
-  curl -o kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.17.9/2020-08-04/bin/linux/amd64/kubectl && \
+  curl -o kubectl kubectl https://s3.us-west-2.amazonaws.com/amazon-eks/1.22.6/2022-03-09/bin/linux/amd64/kubectl && \
```

Comment:
Perhaps I am misreading but should kubectl be here twice?

Reply:
Sorry, fixed. This is what happens when I'm making PR's at midnight.